tests/model_serving/model_runtime/mlserver/negative/test_version_mismatch.py

```
@@ -0,0 +1,75 @@
+        port = Ports.REST_PORT
+
+        with portforward.forward(
+            pod_or_service=mlserver_pod_resource.name,
+            namespace=negative_mlserver_isvc_no_wait.namespace,
+            from_port=port,
+            to_port=port,
+        ):
+            response = requests.post(
+                url=f"{LOCALHOST_URL}:{port}{endpoint}",
+                json=SKLEARN_REST_INPUT_QUERY,
+                verify=False,
```

Ruff (0.15.18)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)